code/src/index.css

```diff
@@ -1,13 +1,275 @@
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background: #000;
+  color: #fff;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+a {
+  color: inherit;
+}
+
+.list {
+  height: 80vh;
+} 
+
+.outerWrapper { 
+  width: 100%;
+}
+
+.innerWrapper {
+  width: 99%;
+  margin: 0 auto;
+}
+
+/* Our mainPage here we show all the movies, and if you hover the image will 
+   turn darker and the name and realease date will show at the bottom */
+
+.sectionWrapper {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.movieBox {
+  width: 25%;
+  size: fit-content;
+  position: relative;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.movieBox .hoverBox {
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  text-align: center;
+}
+
+.hoverDetails {
+  display: none;
+  /* font-size: 180%; */
+}
+
+ .hoverTitle {
+  font-size: 200%;
+  /* margin: 1%; */
+}
+
+.hoverDate {
+  font-size: 130%;
+  margin-bottom: 50%;
+ }
+
+.movieBox:hover .hoverBox {
+   display: flex;
+  flex-direction: column;
+  text-align: center;
+  justify-content: flex-end;
+  background: rgba(0,0,0,.75);
+}
+
+.movieBox:hover .hoverBox .hoverDetails{
+  display: flex;
+ flex-direction: column;
+ text-align: center;
+ justify-content: flex-end;
+}
+
+.img {
+  width: 100%;
+}
+/*back button, The span creates the arrow, and if you hover on it it will move to the left.*/
+.backButton {
+  grid-area: 1 / 1 / span 1 / span 1;
+  display: flex;
+  color: white;
+  background-color: inherit;
+  padding: 10px 16px;
+  border-radius: 20px;
+  transition: all .3s ease;
+  font-weight: bold;
+  cursor: pointer;
+  align-items: center;
+  font-size: 20px;
+  border-style: none;
+}
+
+.backButton > .arrow {
+  width: 30px;
+  height: 30px;
+  border-right: 8px solid white;
+  border-bottom: 8px solid white;
+  position: relative;
+  transform: rotate(135deg);
+  margin: 0 6px;
+  transition: all .3s ease;
+}
+
+.backButton > .arrow::before {
+  display: block;
+  background-color: currentColor;
+  width: 4px;
+  transform-origin: bottom right;
+  height: 3px;
+  position: absolute;
+  opacity: 0;
+  bottom: calc(-2px / 2);
+  transform: rotate(45deg);
+  transition: all .3s ease;
+  content: "";
+  right: 0;
+}
+
+.backButton:hover > .arrow {
+  transform: rotate(135deg) translate(4px, 4px);
+  border-color: text-hover-color;
+}
+
+.backButton:hover > .arrow::before {
+  opacity: 1;
+  width: 20px;
+}
+
+.backButton:hover {
+  background-color: transparent;
+  color: white;
+}
+
+/*Creates the layout and structure for the details*/
+.detailPage {
+  min-height: 100vh;
+  display: flex;
+  background-size: cover;
+  flex-direction: column;
+  justify-content: flex-end;
+}
+
+.detailBox {
+  width: 70%;
+  margin: 0 auto;
+  display: grid; 
+  grid-template-columns: repeat(5, 20%);
+  grid-template-rows: repeat(5, 20%);
```

Comment:
Could you please explain about the reason why you use % instead of fraction? What's the difference?